### ISSUE TITLE
SFC: Support `mixins` and `extends`

### ISSUE THREAD

**Issue by wolfgangwalther** [intlify/eslint-plugin-vue-i18n#109]
Currently the following still throws `error: unused 'test' key (@intlify/vue-i18n/no-unused-keys)` - even if either `OtherComponent` or `Mixin` do use the `test` translation key:

```vue
<i18n lang="yaml" locale="en">
test: Test
</i18n>

<template>
  <div />
</template>

<script>
import OtherComponent from 'OtherComponent'
import Mixin from 'Mixin'

export default {
  extends: OtherComponent,
  mixins: [Mixin]
}
</script>
```

I would expect this to pass if the key is used in any inherited component/mixin.

It would be cool if that worked the other way around as well (so the `<i18n>` block in the OtherComponent or Mixin - and the usage of the translation keys in this component) for the `no-missing-keys` rule. (I don't know whether that actually works right now with vue-i18n at all. I have a custom merge strategy in place, that merges i18n blocks with those extends/mixin components, so it does for me quite well.)

**Comment by ota-meshi** [comment]
Thank you for this issue.

`mixins` analysis can be very time consuming, so this plugin does not support it.

This plugin has some exceptions, but ESLint basically only provides rules that can be parsed in one file. Therefore, it does not have a way to parse multiple files centrally. This limitation applies to ESLint plugin as well.

Thank you for your understanding.
